manifests/init.pp

```diff
@@ -189,6 +192,7 @@
           $default_pid_file        = '/var/run/syslogd.pid'
           $sysconfig_erb           = 'sysconfig.rhel7.erb'
           $default_syslogd_options = '-c 4'
+          $mod_imjournal           = true
```

Comment:
A bit confused here.  Where are you thinking that ```$default_module_imjournal``` should be set?  I'm assuming that the "default" should be ```false``` since this is only required on RHEL7?